demos/mlsys_2020/demo_handler.py

```diff
@@ -0,0 +1,137 @@
+from demo_source import *
+
+myDemo = Demo()
+
+# Functional widget: Text Box
+op_textbox = lambda x: widgets.Text(value=list(myDemo.edsl_snippets[x].values())[0],
```

Comment:
I think we should try to pick one naming scheme, like `snake_case` or `camelCase` but not a mix of both.